db/version_set.cc

```diff
@@ -7456,6 +7457,9 @@ Status ReactiveVersionSet::ReadAndApply(
   if (s.ok()) {
     *cfds_changed = std::move(manifest_tailer_->GetUpdatedColumnFamilies());
   }
+  if (files_to_delete) {
+    *files_to_delete = std::move(manifest_tailer_->GetFilesToDelete());
```

Comment:
nit: does this move actually move the vector, is this intended to be an ownership transfer?

Reply:
Yes, we want to transfer ownership, which will clear the original vector.

Reply:
Ah, that' very neat! Does `GetFilesToDelete()` function need to return a rvalue reference for this to work?